### PR TITLE
fix(293338): fix changeLog file url on PR template

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue: #

## Proposed Changes

 - [x] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Description

fix changeLog file url on PR template

## Impact on version

- [ ] **Major**
  - The template structure was changed.
- [ ] **Minor**
  - New functionalities were added.
- [x] **Patch**
  - A bug in behavior was fixed.
  - Documentation was changed.

### Always applicable
No matter your changes, these checks always apply.
- [x] Your conventional commits are aligned with the **Impact on version** section.
- [ ] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [ ] Documentation files were updated according with the changes.
  - Update `README.md` and `TemplateConfig.md` if you made changes to **templating**.
  - Update `AzurePipelines.md` and `APP_README.md` if you made changes to **pipelines**.
  - Update `Diagnostics.md` if you made changes to **diagnostic tools**.
  - Update `Architecture.md` and its diagrams if you made **architecture decisions** or if you introduced new **recipes**.
  - ...and so forth: Make sure you update the documentation files associated to the recipes you changed. Review the topics by looking at the content of the `doc/` folder.
- [ ] Images about the template are referenced from the [wiki](https://github.com/nventive/UnoApplicationTemplate/wiki/Images) and added as images in this git.
- [ ] TODO comments are hints for next steps for users of the template and not planned work.

### Contextual
Based on your changes these checks may not apply.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Tested on all relevant platforms

## Other information

NA

## Internal Issue (If applicable):

NA

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ GitHub Issue: #
 ### Always applicable
 No matter your changes, these checks always apply.
 - [ ] Your conventional commits are aligned with the **Impact on version** section.
-- [ ] Updated [CHANGELOG.md](../CHANGELOG.md).
+- [ ] Updated [CHANGELOG.md](../blob/main/CHANGELOG.md).
```
Comment (jeanplevesque) [inline .github/PULL_REQUEST_TEMPLATE.md:35]: I'm not sure this works? I tested by trying the link in your branch directly.

Comment (Arieldelossantos) [reply .github/PULL_REQUEST_TEMPLATE.md:35]: uhmm... how i did it was comparing to the changelog file link and the link from the current PR template and that was the only part missing in the URL :/ 
the weird part is that those folders don't exists in the proj 
@jeanplevesque

Comment (Marc-Antoine-Soucy) [reply .github/PULL_REQUEST_TEMPLATE.md:35]: I think this is just saying that we're looking at the changelog of the main branch, and this change does work you can look at this draft pr: https://github.com/nventive/UnoApplicationTemplate/pull/409 I applied the changes from Ariel and it leads you to the correct place.

**Review by Marc-Antoine-Soucy — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jeanplevesque) [reply .github/PULL_REQUEST_TEMPLATE.md:35]: ok

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**